api/lib/MulterConfig.js

```diff
@@ -0,0 +1,31 @@
+const multer  = require('multer');
+const httpStatus = require('./http-status-codes');
+
+const configs = {
+    menus: {
+        fileFilter: (req, file, cb) => {
+            if (file.mimetype !== 'application/pdf'){
+                let err = new Error(`Wrong file type: "${file.mimetype}"`);
+                err.status = httpStatus.BAD_REQUEST;
+                return cb(err, false);
+            }
+            return cb(null, true);
+        },
+        limits: {
+            fileSize: 2097152, // 2MB
```

Comment:
@gabrieldrs I don't know if is an issue or not, but I saw that 40% of our sample pass than 2MB and it's directly related with their quality... Do you think is better increase the file Size limit?

Reply:
Sure, by how much do you think we should limit it?

Reply:
I've just checked and the largest one that we have is "ks-prime-sushi-bar" with 11MB.

I do think 11MB is too much for a single file in our case, so I will limit to 10MB, and I will consider this specific menu as an outlier.